Resources/Prototypes/Entities/Objects/Tools/glowstick.yml

```diff
@@ -13,26 +13,27 @@
       fadeOutDuration: 300
       turnOnBehaviourID: turn_on
       fadeOutBehaviourID: fade_out
+      iconStateLit: glowstick_lit
+      iconStateSpent: glowstick_unlit
       litSound:
         path: /Audio/Items/Handcuffs/rope_breakout.ogg
     - type: Sprite
       sprite: Objects/Misc/glowstick.rsi
       layers:
-        - state: glowstick_base
-        - state: glowstick_glow
-          color: "#00FF00"
-          visible: false
-          shader: unshaded
-        - state: glowstick_unlit
-          color: "#00FF00"
+      - map: [ enum.ExpendableLightVisualLayers.Base ]
+        state: glowstick_base
+      - map: [ enum.ExpendableLightVisualLayers.Glow ]
+        state: glowstick_glow
+        color: "#00FF00"
+        visible: false
+        shader: unshaded
+      - map: [ enum.ExpendableLightVisualLayers.Overlay ]
+        state: glowstick_unlit
+        color: "#00FF00"
```

Comment:
make this map color part of the component so you can just inherit instead of repeating so much imo

Reply:
Done.